images/bootstrap/runner_wrapper.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright 2020 The KubeVirt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+setup_env(){
+    if [ ! -z "${DOCKER_HTTP_PROXY}" ]; then
+        export HTTP_PROXY="${DOCKER_HTTP_PROXY}"
+    fi
+
+    if [ ! -z "${DOCKER_HTTPS_PROXY}" ]; then
+        export HTTPS_PROXY="${DOCKER_HTTPS_PROXY}"
+    fi
+}
+
+setup_ca(){
+    if [ -f "${CA_CERT_FILE}" ]; then
+        cp "${CA_CERT_FILE}" /usr/local/share/ca-certificates/
+
+        update-ca-certificates
+    fi
+}
+
+main(){
+    setup_env
```

Comment:
I think that there is a debian service for docker in that image. See `service docker start` there. I think we only want to inject the proxies for this service and not for anything else which happens in there.